app/models/photo.rb

```diff
@@ -96,26 +96,24 @@ def processed?
   end
 
   def update_remote_path
-    unless self.unprocessed_image.url.match(/^https?:\/\//)
-      remote_path = "#{AppConfig.pod_uri.to_s.chomp("/")}#{self.unprocessed_image.url}"
-    else
-      remote_path = self.unprocessed_image.url
-    end
+    remote_path = if unprocessed_image.url.match(%r{^https?://})
+                    unprocessed_image.url
+                  else
+                    "#{AppConfig.pod_uri.to_s.chomp('/')}#{unprocessed_image.url}"
+                  end
 
-    name_start = remote_path.rindex '/'
+    name_start = remote_path.rindex "/"
     self.remote_photo_path = "#{remote_path.slice(0, name_start)}/"
     self.remote_photo_name = remote_path.slice(name_start + 1, remote_path.length)
   end
 
-  def url(name = nil)
-    if remote_photo_path
-      name = name.to_s + '_' if name
+  def url(name=nil)
+    # During migration in bad cases this might be happen
```

Comment:
Hm - I think I discovered this once. Leave it or remove it?

Reply:
You can leave the double checks, since I don't think it harms anything ... but this comment confuses me more than it helps. It doesn't say "what" maybe happens (which maybe is less a problem with the context of this diff, but if you look at the code only, I have no idea what the problem should be. So I guess you mean `remote_photo_name` could be nil while `remote_photo_path` is set? But that should never happen, they should always be set together, only setting one makes no sense. But also `remote_photo_name` should also never be nil in an import anyway, since it's a required field? So I say, just remove the comment, the code is fine as it was but I also think it's fine with the second check now.